test/member.spec.ts

```diff
@@ -100,4 +100,15 @@ describe('Member', () => {
     expect(votes[votes.length - 1].length).toBeGreaterThan(0)
     expect(votes[votes.length - 1].map((vote) => vote.proposalId)).toContain(proposal.id)
   })
+
+  it('Members are searchable', async () => {
+    let members: Member[] = []
+
+    Member.search({}, arc)
+      .subscribe((result) => members = result)
+
+    await waitUntilTrue(() => members.length !== 0)
+
+    expect(members.length).toEqual(15)
```

Comment:
we cannot really be sure this will be 15, as another test might distribute some new repuation to a new account (and thus add a member to the collection). Can you change this to toBeGreaterThanOrEqual ?